spec/normalize.js

```diff
@@ -6,7 +6,9 @@ console.log('node path changed');
 
 beforeEach(function(){
     browser.driver.manage().window().maximize();
-
+    browser.getCapabilities().then(function(cap) {
+        browser.browserName = cap.get('browserName');
```

Comment:
is it a good practice putting the browserName on the global? (browser), I think I used to get it from an env variable